README.md

```diff
@@ -1,13 +1,82 @@
-# Metadev-Client-Application-Example
+Metadev-Client-Application-Example
+===========
 
-## Installing Docker 
+## Introduction.
 
-Follow [docker docs](https://docs.docker.com/get-docker/) to install docker.
+Metadev-client-example is an example project which uses the Metadev client framework. The goal of the framework is to reduce the code written for frontend development. The framework is best suited for [ERP](https://en.wikipedia.org/wiki/Enterprise_resource_planning) apps which have a ton of forms.
+
+
+**Metadev** is an abbreviation for Metadata driven development.
+
+---
+
+## How it works?
+
+The project is built using Typescript and Angular. Also the framework uses [Material Design Components](https://material.angular.io/). 
+
+[Metadev-Server-Example](https://github.com/supreethavadhani/metadev-server-example) generates the models using json. The generated files are then copied into to the project.
+*The generated files can be found [here](https://github.com/supreethavadhani/metadev-client-example/tree/master/src/app/framework-modules/formdata/gen).*
+
+The metadev-client framework is built into the project for this version and can be found [here](https://github.com/supreethavadhani/metadev-client-example/tree/master/src/app/framework-modules).
+
+##### *In the next version the `framework-modules` folder will be packaged and moved into a entirely new project.*
+
+---
+
+## Example.
+
+A frontend developer only needs to write a single line of html code to generate forms.
```

Comment:
Possible elaboration here: "This code displays the Metadev Angular component, which generates the form based on the JSON files provided."

(If I understand how this works correctly.)

Reply:
You understood it perfectly. Will make the change 💯